readme.md

```diff
@@ -164,20 +164,47 @@ Note:
 
 For more details please refer [these](https://github.com/Azure/autorest/tree/master/docs/generate) docs to generate code from your OpenAPI definition using AutoRest.
 
-## Debugging 
+## Debugging
 
-There are two entry points for debugging autorest.csharp:
+### Debugging projects inside the `autorest.csharp` repo
 
-- Opening AutoRest.CSharp.sln in Visual Studio will contain a debugging profile for each sample and test. These only run the "back half", using the yaml file generated by autorest.core as input.
-- ./Generate.ps1 will execute the entire pipeline by invoking autorest and including autorest.csharp as a plugin. Add:
+There are two entry points for debugging the generator:
```

Comment:
There is another way to debug typespec project inside `autorest.csharp`:
```powershell
dotnet run --project ${path_to_Autorest.CSharp_project_folder} --standalone ${path_to_generated_folder_of_target_project} --debug
```

or
```powershell
dotnet run ${path_to_Autorest.CSharp_dll} --standalone ${path_to_generated_folder_of_target_project} --debug
```

This actually works for project outside of `autorest.csharp`, as long as `tspCodeModel.json` and `Configuration.json` are available.

Reply:
This is hacky, I prefer we do not document them here.
Debugging inside the Visual Studio should be enough.